tests/test_series.py

```diff
@@ -3069,3 +3069,15 @@ def first_arg_not_series(argument_1: int, ser: pd.Series) -> pd.Series:
             ),
             1,
         )
+
+
+def test_series_apply() -> None:
+    s = pd.Series(["A", "B", "AB"])
+    s.apply(tuple)
+    check(assert_type(s, "pd.Series[str]"), pd.Series, tuple)
```

Comment:
That is testing `s`, but it should be testing the return value of `s.apply`.

Reply:
That makes sense but I can't seem to get it to work. 

I am getting errors when I do this:
check(assert_type(s.apply(tuple), "pd.Series[tuple]"), pd.Series) 
The error I'm getting is: 
![image](https://github.com/pandas-dev/pandas-stubs/assets/132183038/601536db-55b0-4abe-93cb-7e6bf3169e9e)

Any tips?

Reply:
I think you might have to use `"pd.Series[Any]"`

Reply:
Fixed it!

Reply:
`black` and then it should be ready

Reply:
When I run "pre-commit run --all-files" locally, black passes. Can't seem to find the issue.

Reply:
Strange, there could be a discrepancy between the black in pyproject and .pre-commit but when running pre-commit it should always use the one from .pre-commit (which should be the same version that the CI uses).

Reply:
My local was automatically fixed when running pre-commit and I didn't notice that it had changed the file to fix the black error.